.travis.yml

```diff
@@ -5,6 +5,7 @@ before_deploy:
   - rustup target add aarch64-apple-ios armv7-apple-ios i386-apple-ios x86_64-apple-ios
   - cargo install cargo-lipo
   - brew update && (brew outdated carthage || brew upgrade carthage)
+  - brew install protobuf
```

Comment:
in my case locally i did a 
```
brew list protobuf             
/usr/local/Cellar/protobuf/3.5.1_1/bin/protoc
/usr/local/Cellar/protobuf/3.5.1_1/include/google/ (126 files)
/usr/local/Cellar/protobuf/3.5.1_1/lib/libprotobuf-lite.15.dylib
/usr/local/Cellar/protobuf/3.5.1_1/lib/libprotobuf.15.dylib
/usr/local/Cellar/protobuf/3.5.1_1/lib/libprotoc.15.dylib
/usr/local/Cellar/protobuf/3.5.1_1/lib/pkgconfig/ (2 files)
/usr/local/Cellar/protobuf/3.5.1_1/lib/python2.7/site-packages/homebrew-protobuf.pth
/usr/local/Cellar/protobuf/3.5.1_1/lib/ (6 other files)
/usr/local/Cellar/protobuf/3.5.1_1/libexec/lib/ (104 files)
/usr/local/Cellar/protobuf/3.5.1_1/share/doc/ (19 files)
```
and it still seems to work? so that's good

![](http://v14d.com/i/25c38ea553bfe416e6320e8ef8b22095fd7a6ab2de62519da9e295d2b3c0b98f.jpg)

Reply:
This is fine as the underlying protocols don't change.